Makefile

```diff
@@ -47,3 +47,17 @@ push-aur:
 clean:
 	rm -f $(MANS)
 	rm -f sbctl
+
+.PHONY: deps
+deps:
+	go get honnef.co/go/tools/cmd/staticcheck@2020.1.6
+
+.PHONY: lint
+lint: deps
+	go vet ./...
+	test -z $(shell go fmt ./...)
```

Comment:
You might want to quote this, I'm not sure how it behaves in make.

Reply:
I'm not actually sure this is the best method of doing this, so I might revert.

Reply:
`go fmt` annoyingly doesn't return error codes.